docs/superbench-config.mdx

```diff
@@ -209,6 +209,7 @@ ${benchmark_name}:
   parameters:
     run_count: int
     duration: int
```

Comment:
do we need to remove `duration` in micro-benchmark?

Reply:
I think we can keep it in case some micro-benchmark can use it to control the execution time by itself, what do you think?